config/default.yaml

```diff
@@ -95,8 +95,8 @@ executor:
             # The host or IP of the kubernetes cluster
             host: kubernetes.default
             # The jwt token used for authenticating kubernetes requests
-            # Loaded from /var/run/secrets/kubernetes.io/serviceaccount/token by default
-            # token:
+            # Loaded from /var/run/secrets/kubernetes.io/serviceaccount/token by default if not specified
+            token: 'fakeToken'
```

Comment:
The issue we're going to have is when we run this in the hosted cluster. We want the cluster to read from `/var/run/secrets/kubernetes.io/serviceaccount/token`.

Doesn't this change mean we have to now add another configuration change to override this fake token?